bootstrap/cmd/bootstrap/app/gcpUtils.go

```diff
@@ -58,36 +58,34 @@ func init() {
 }
 
 // TODO: handle concurrent & repetitive deployment requests.
-func (s *ksServer) InsertDeployment(ctx context.Context, req CreateRequest) error {
+func (s *ksServer) InsertDeployment(ctx context.Context, req CreateRequest) (*deploymentmanager.Deployment, error) {
 	regPath := s.knownRegistries["kubeflow"].RegUri
 	var dmconf DmConf
 	err := LoadConfig(path.Join(regPath, "../deployment/gke/deployment_manager_configs/cluster-kubeflow.yaml"), &dmconf)
 
 	if err == nil {
 		dmconf.Resources[0].Name = req.Name
 		dmconf.Resources[0].Properties["zone"] = req.Zone
-		dmconf.Resources[0].Properties["clientId"] = req.ClientId
-		dmconf.Resources[0].Properties["clientSecret"] = req.ClientSecret
 		dmconf.Resources[0].Properties["ipName"] = req.IpName
-		dmconf.Resources[0].Properties["isWebapp"] = true
+		dmconf.Resources[0].Properties["gpu-pool-max-nodes"] = 10
```

Comment:
Lets default this to 0 for now. What we should do is file a follow on issue to automatically check whether the user has GPU quota in the zone and if they do set it.